components/channel/ChannelParameterUnwrap.js

```diff
@@ -18,8 +20,8 @@ import { camelCase, castToTsType, realizeChannelNameWithoutParameters } from '..
   const streetlightIdParam = "" + channel.substring(0, streetlightIdEnd);
  * 
  * 
- * @param {*} channelName to be unwrapped
- * @param {*} channelParameters the parameters which are to be unwrapped from the NATS topic.
+ * @param {string} channelName to be unwrapped
+ * @param {{[key: string]: ChannelParameter}} channelParameters the parameters which are to be unwrapped from the NATS topic.

```

Comment:
```suggestion
 * @param {Object.<string, ChannelParameter>} channelParameters the parameters which are to be unwrapped from the NATS topic.
```

```
Object.<key, value>
```

is a proper way to define dictionary in JSDoc

Reply:
Did not know that, thanks! I'll do a search/replace 😄